x-pack/legacy/plugins/integrations_manager/public/components/integration_list_grid.tsx

```diff
@@ -4,21 +4,66 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
-import { EuiFlexGrid, EuiFlexItem } from '@elastic/eui';
-import { IntegrationList, IntegrationListItem } from '../../common/types';
+import { EuiFlexGrid, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText } from '@elastic/eui';
+import { STATUS_INSTALLED, STATUS_NOT_INSTALLED } from '../../common/constants';
+import {
+  InstallationStatus,
+  IntegrationsGroupedByStatus,
+  IntegrationList,
+  IntegrationListItem,
+} from '../../common/types';
 import { IntegrationCard } from './integration_card';
 
+interface GridProps {
+  map: IntegrationsGroupedByStatus;
+}
+
 interface ListProps {
+  status: InstallationStatus;
   list: IntegrationList;
 }
 
-export function IntegrationListGrid({ list }: ListProps) {
+// Calling Object.entries(IntegrationsGroupedByStatus) gave `status: string`
+// which causes a "string is not assignable to type InstallationStatus` error
+// see https://github.com/Microsoft/TypeScript/issues/20322
+// and https://github.com/Microsoft/TypeScript/pull/12253#issuecomment-263132208
+const entries = Object.entries as <T>(o: T) => Array<[Extract<keyof T, string>, T[keyof T]]>;
+
+export function IntegrationsGridByStatus({ map }: GridProps) {
+  if (!map) return null;
   return (
-    <EuiFlexGrid gutterSize="l" columns={3}>
-      {list.map(item => (
-        <GridItem key={`${item.name}-${item.version}`} {...item} />
+    <>
```

Comment:
Do you know if we use this syntax anywhere else? I don't mind it honestly but usually I see `<React.Fragment>`

Reply:
Looks like both are used. Even within the same plugin. <details><summary><code>ack -lc '<>' x-pack/legacy/plugins/</code></summary>
x-pack/legacy/plugins/infra/docs/arch_client.md:4
x-pack/legacy/plugins/infra/public/components/source_configuration/source_configuration_flyout.tsx:3
x-pack/legacy/plugins/infra/public/components/source_configuration/log_columns_configuration_panel.tsx:2
x-pack/legacy/plugins/infra/public/components/logging/log_text_stream/scrollable_log_text_stream_view.tsx:2
x-pack/legacy/plugins/infra/public/components/logging/log_minimap/search_marker.tsx:2
x-pack/legacy/plugins/infra/public/containers/logs/log_highlights/redux_bridges.tsx:1
x-pack/legacy/plugins/infra/public/pages/metrics/index.tsx:1
x-pack/legacy/plugins/infra/public/pages/logs/page_logs_content.tsx:1
x-pack/legacy/plugins/infra/public/pages/logs/page_header.tsx:1
x-pack/legacy/plugins/infra/public/pages/logs/page_content.tsx:1
x-pack/legacy/plugins/infra/public/pages/infrastructure/snapshot/index.tsx:1
x-pack/legacy/plugins/apm/public/components/app/Settings/SettingsList.tsx:2
x-pack/legacy/plugins/apm/public/components/app/Settings/AddSettings/AddSettingFlyoutBody.tsx:1
x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx:2
x-pack/legacy/plugins/apm/public/components/shared/ApmHeader/index.tsx:1
x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/index.tsx:1
x-pack/legacy/plugins/apm/public/components/shared/EnvironmentBadge/index.tsx:1
x-pack/legacy/plugins/integrations_manager/public/screens/home.tsx:1
x-pack/legacy/plugins/integrations_manager/public/components/integration_list_grid.tsx:2
x-pack/legacy/plugins/security/public/components/session_expiration_warning/session_expiration_warning.tsx:1
x-pack/legacy/plugins/security/public/lib/validate_user.ts:1
x-pack/legacy/plugins/security/public/views/management/edit_role/components/privileges/es/index_privilege_form.tsx:1
x-pack/legacy/plugins/security/public/views/management/edit_role/lib/validate_role.test.ts:1
x-pack/legacy/plugins/spaces/public/views/management/lib/space_identifier_utils.test.ts:1
x-pack/legacy/plugins/code/common/repository_utils.ts:1
x-pack/legacy/plugins/siem/public/components/with_hover_actions/index.tsx:2
x-pack/legacy/plugins/siem/public/components/open_timeline/delete_timeline_modal/index.tsx:1
x-pack/legacy/plugins/siem/public/components/open_timeline/search_row/index.tsx:1
x-pack/legacy/plugins/siem/public/components/open_timeline/open_timeline_modal/index.tsx:1
x-pack/legacy/plugins/siem/public/components/open_timeline/open_timeline_modal/open_timeline_modal.tsx:1
x-pack/legacy/plugins/siem/public/components/stat_items/index.test.tsx:3
x-pack/legacy/plugins/siem/public/components/draggables/index.tsx:2
x-pack/legacy/plugins/siem/public/components/localized_date_tooltip/index.tsx:1
x-pack/legacy/plugins/siem/public/components/tables/helpers.tsx:8
x-pack/legacy/plugins/siem/public/components/tables/helpers.test.tsx:2
x-pack/legacy/plugins/siem/public/components/page/network/domains_table/columns.tsx:3
x-pack/legacy/plugins/siem/public/components/page/network/network_top_n_flow_table/columns.tsx:1
x-pack/legacy/plugins/siem/public/components/page/hosts/events_table/index.tsx:3
x-pack/legacy/plugins/siem/public/components/page/hosts/first_last_seen_host/index.tsx:1
x-pack/legacy/plugins/siem/public/components/page/hosts/uncommon_process_table/index.tsx:1
x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/columns.tsx:3
x-pack/legacy/plugins/siem/public/components/page/overview/overview_host_stats/index.tsx:1
x-pack/legacy/plugins/siem/public/components/page/overview/overview_network_stats/index.tsx:1
x-pack/legacy/plugins/siem/public/components/page/add_to_kql/index.test.tsx:4
x-pack/legacy/plugins/siem/public/components/empty_value/index.tsx:3
x-pack/legacy/plugins/siem/public/components/resize_handle/index.tsx:1
x-pack/legacy/plugins/siem/public/components/resize_handle/index.test.tsx:5
x-pack/legacy/plugins/siem/public/components/inspect/modal.tsx:3
x-pack/legacy/plugins/siem/public/components/formatted_date/index.tsx:1
x-pack/legacy/plugins/siem/public/components/formatted_ip/index.tsx:1
x-pack/legacy/plugins/siem/public/components/formatted_bytes/index.tsx:1
x-pack/legacy/plugins/siem/public/components/paginated_table/index.tsx:1
x-pack/legacy/plugins/siem/public/components/fields_browser/index.tsx:1
x-pack/legacy/plugins/siem/public/components/fields_browser/category.tsx:1
x-pack/legacy/plugins/siem/public/components/fields_browser/categories_pane.tsx:1
x-pack/legacy/plugins/siem/public/components/fields_browser/field_items.tsx:1
x-pack/legacy/plugins/siem/public/components/fields_browser/fields_pane.tsx:1
x-pack/legacy/plugins/siem/public/components/field_renderers/field_renderers.tsx:8
x-pack/legacy/plugins/siem/public/components/toasters/index.tsx:1
x-pack/legacy/plugins/siem/public/components/toasters/index.test.tsx:7
x-pack/legacy/plugins/siem/public/components/formatted_duration/tooltip/index.tsx:2
x-pack/legacy/plugins/siem/public/components/load_more_table/index.tsx:1
x-pack/legacy/plugins/siem/public/components/help_menu/help_menu.tsx:1
x-pack/legacy/plugins/siem/public/components/ml/anomaly/anomaly_table_provider.tsx:1
x-pack/legacy/plugins/siem/public/components/ml/score/create_description_list.tsx:2
x-pack/legacy/plugins/siem/public/components/ml/score/anomaly_score.tsx:1
x-pack/legacy/plugins/siem/public/components/ml/score/anomaly_scores.tsx:1
x-pack/legacy/plugins/siem/public/components/ml/score/draggable_score.tsx:2
x-pack/legacy/plugins/siem/public/components/ml/entity_draggable.tsx:1
x-pack/legacy/plugins/siem/public/components/last_event_time/index.tsx:1
x-pack/legacy/plugins/siem/public/components/lazy_accordion/index.tsx:3
x-pack/legacy/plugins/siem/public/components/source_destination/geo_fields.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/footer/last_updated.tsx:2
x-pack/legacy/plugins/siem/public/components/timeline/footer/index.tsx:3
x-pack/legacy/plugins/siem/public/components/timeline/refetch_timeline.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/user_host_working_dir.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/process_draggable.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/suricata/suricata_signature.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/system/package.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/system/generic_file_details.tsx:2
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/system/generic_details.tsx:2
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/system/auth_ssh.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/formatted_field.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/zeek/zeek_signature.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/auditd/session_user_host_working_dir.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/host_working_dir.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/header_tooltip_content/index.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/header/index.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/body/events/stateful_event.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/auto_save_warning/index.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/properties/helpers.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/data_providers/provider_badge.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/data_providers/empty.tsx:1
x-pack/legacy/plugins/siem/public/components/timeline/search_or_filter/helpers.tsx:2
x-pack/legacy/plugins/siem/public/components/header_page/header_page.tsx:1
x-pack/legacy/plugins/siem/public/components/flyout/index.tsx:1
x-pack/legacy/plugins/siem/public/components/flyout/pane/index.tsx:1
x-pack/legacy/plugins/siem/public/lib/keury/index.ts:1
x-pack/legacy/plugins/siem/public/lib/clipboard/with_copy_to_clipboard.tsx:1
x-pack/legacy/plugins/siem/public/containers/network/filter.tsx:1
x-pack/legacy/plugins/siem/public/containers/hosts/filter.tsx:1
x-pack/legacy/plugins/siem/public/containers/global_time/index.tsx:1
x-pack/legacy/plugins/siem/public/pages/network/index.tsx:1
x-pack/legacy/plugins/siem/public/pages/network/ip_details.tsx:2
x-pack/legacy/plugins/siem/public/pages/network/network.tsx:2
x-pack/legacy/plugins/siem/public/pages/timelines/timelines_page.tsx:1
x-pack/legacy/plugins/siem/public/pages/hosts/index.tsx:1
x-pack/legacy/plugins/siem/public/pages/hosts/hosts.tsx:2
x-pack/legacy/plugins/siem/public/pages/hosts/host_details.tsx:2
x-pack/legacy/plugins/siem/public/pages/overview/overview.tsx:1
x-pack/legacy/plugins/kuery_autocomplete/public/autocomplete_providers/escape_kuery.js:1
x-pack/legacy/plugins/ml/common/util/es_utils.ts:1
x-pack/legacy/plugins/ml/public/util/__tests__/custom_url_utils.js:3
x-pack/legacy/plugins/ml/public/util/string_utils.js:2
x-pack/legacy/plugins/ml/public/components/kql_filter_bar/__tests__/utils.js:3
x-pack/legacy/plugins/monitoring/public/components/cluster/overview/elasticsearch_panel.js:1
x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/validators/validate_name.test.js:1</details>

<details><summary><code>ack -lc 'React.Fragment' x-pack/legacy/plugins/</code></summary>
x-pack/legacy/plugins/maps/public/components/map_listing.js:4
x-pack/legacy/plugins/maps/public/components/__snapshots__/layer_toc_actions.test.js.snap:6
x-pack/legacy/plugins/infra/public/components/metrics/index.tsx:3
x-pack/legacy/plugins/infra/public/pages/infrastructure/snapshot/toolbar.tsx:2
x-pack/legacy/plugins/infra/public/pages/infrastructure/metrics_explorer/index.tsx:2
x-pack/legacy/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/health.tsx:4
x-pack/legacy/plugins/upgrade_assistant/public/components/tabs/checkup/__snapshots__/checkup_tab.test.tsx.snap:2
x-pack/legacy/plugins/apm/public/context/UrlParamsContext/__tests__/UrlParamsContext.test.tsx:4
x-pack/legacy/plugins/apm/public/components/app/TransactionOverview/index.tsx:2
x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/__test__/__snapshots__/NoServicesMessage.test.tsx.snap:2
x-pack/legacy/plugins/apm/public/components/app/ServiceOverview/NoServicesMessage.tsx:2
x-pack/legacy/plugins/apm/public/components/app/ServiceMetrics/index.tsx:2
x-pack/legacy/plugins/apm/public/components/app/ServiceMetrics/MetricsChart.tsx:2
x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/TransactionFlyout/DroppedSpansWarning.tsx:2
x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/TransactionTabs.tsx:2
x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/index.tsx:2
x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/index.tsx:4
x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx:4
x-pack/legacy/plugins/apm/public/components/shared/DottedKeyValueTable/FormattedValue.tsx:3
x-pack/legacy/plugins/apm/public/components/shared/HistoryTabs/index.tsx:2
x-pack/legacy/plugins/apm/public/components/shared/EnvironmentBadge/index.tsx:2
x-pack/legacy/plugins/beats_management/public/components/enroll_beats.tsx:4
x-pack/legacy/plugins/beats_management/public/components/navigation/child_routes.tsx:1
x-pack/legacy/plugins/beats_management/public/components/layouts/primary.tsx:2
x-pack/legacy/plugins/beats_management/public/router.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/beat/index.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/beat/details.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/beat/tags.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/walkthrough/initial/tag.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/walkthrough/initial/finish.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/walkthrough/initial/beat.tsx:4
x-pack/legacy/plugins/beats_management/public/pages/overview/index.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/overview/configuration_tags.tsx:2
x-pack/legacy/plugins/beats_management/public/pages/overview/enrolled_beats.tsx:6
x-pack/legacy/plugins/code/public/components/hover/hover_buttons.tsx:2
x-pack/legacy/plugins/code/public/components/hover/hover_widget.tsx:1
x-pack/legacy/plugins/code/public/components/admin_page/setup_guide.tsx:4
x-pack/legacy/plugins/code/public/components/admin_page/language_server_tab.tsx:2
x-pack/legacy/plugins/code/public/components/help_menu/help_menu.tsx:2
x-pack/legacy/plugins/code/public/components/shortcuts/shortcuts_provider.tsx:2
x-pack/legacy/plugins/code/public/components/main/main.tsx:2
x-pack/legacy/plugins/code/public/components/main/content.tsx:2
x-pack/legacy/plugins/code/public/components/main/directory.tsx:2
x-pack/legacy/plugins/code/public/components/diff_page/diff.tsx:2
x-pack/legacy/plugins/code/public/components/editor/references_panel.tsx:2
x-pack/legacy/plugins/siem/public/components/tables/helpers.tsx:2
x-pack/legacy/plugins/siem/public/components/tables/__snapshots__/helpers.test.tsx.snap:4
x-pack/legacy/plugins/siem/public/components/page/network/kpi_network/index.tsx:2
x-pack/legacy/plugins/siem/public/components/page/overview/overview_host_stats/index.tsx:2
x-pack/legacy/plugins/siem/public/components/page/overview/overview_network_stats/index.tsx:2
x-pack/legacy/plugins/siem/public/components/ml/score/__snapshots__/anomaly_score.test.tsx.snap:4
x-pack/legacy/plugins/siem/public/components/timeline/data_providers/provider_item_and.tsx:2
x-pack/legacy/plugins/ml/public/settings/calendars/list/header.js:2
x-pack/legacy/plugins/ml/public/settings/calendars/list/table/table.js:2
x-pack/legacy/plugins/ml/public/settings/filter_lists/edit/header.js:6
x-pack/legacy/plugins/ml/public/settings/filter_lists/edit/toolbar.js:2
x-pack/legacy/plugins/ml/public/settings/filter_lists/list/header.js:2
x-pack/legacy/plugins/ml/public/settings/filter_lists/list/table.js:6
x-pack/legacy/plugins/ml/public/explorer/explorer_charts/explorer_chart_info_tooltip.js:4
x-pack/legacy/plugins/ml/public/explorer/explorer_charts/components/explorer_chart_label/explorer_chart_label.test.js:1
x-pack/legacy/plugins/ml/public/explorer/explorer_charts/components/explorer_chart_label/explorer_chart_label.js:4
x-pack/legacy/plugins/ml/public/explorer/explorer_charts/explorer_charts_container.js:5
x-pack/legacy/plugins/ml/public/explorer/components/explorer_no_results_found/explorer_no_results_found.js:2
x-pack/legacy/plugins/ml/public/explorer/components/explorer_no_results_found/__snapshots__/explorer_no_results_found.test.js.snap:2
x-pack/legacy/plugins/ml/public/explorer/explorer.js:4
x-pack/legacy/plugins/ml/public/components/upgrade/upgrade_warning.js:2
x-pack/legacy/plugins/ml/public/components/entity_cell/entity_cell.js:2
x-pack/legacy/plugins/ml/public/components/validate_job/validate_job_view.js:4
x-pack/legacy/plugins/ml/public/components/form_label/form_label.js:2
x-pack/legacy/plugins/ml/public/components/anomalies_table/influencers_cell.js:4
x-pack/legacy/plugins/ml/public/components/anomalies_table/detector_cell.js:2
x-pack/legacy/plugins/ml/public/components/anomalies_table/anomalies_table.js:2
x-pack/legacy/plugins/ml/public/components/anomalies_table/anomaly_details.js:8
x-pack/legacy/plugins/ml/public/components/influencers_list/influencers_list.js:4
x-pack/legacy/plugins/ml/public/components/rule_editor/scope_section.js:4
x-pack/legacy/plugins/ml/public/components/rule_editor/select_rule_action/select_rule_action.js:4
x-pack/legacy/plugins/ml/public/components/rule_editor/select_rule_action/delete_rule_modal.js:2
x-pack/legacy/plugins/ml/public/components/rule_editor/rule_editor_flyout.js:2
x-pack/legacy/plugins/ml/public/components/rule_editor/actions_section.js:2
x-pack/legacy/plugins/ml/public/components/rule_editor/conditions_section.js:2
x-pack/legacy/plugins/ml/public/jobs/components/custom_url_editor/list.js:2
x-pack/legacy/plugins/ml/public/jobs/components/custom_url_editor/editor.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/jobs_list_view/jobs_list_view.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/delete_job_modal/delete_job_modal.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/job_actions/results.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/tabs/job_details.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/tabs/custom_urls.js:6
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/tabs/datafeed.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/edit_job_flyout/tabs/detectors.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/create_watch_flyout/create_watch_flyout.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/node_available_warning/node_available_warning.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/job_details/datafeed_preview_tab.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/job_details/json_tab.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/job_details/job_messages_pane.js:2
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/job_details/job_details_pane.js:4
x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/jobs_list/job_description.js:2
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/import_view/import_view.js:8
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/import_errors/errors.js:2
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/import_summary/import_summary.js:4
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/file_contents/file_contents.js:2
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/file_datavisualizer_view/file_datavisualizer_view.js:8
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/import_settings/advanced.js:8
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/import_settings/import_settings.js:6
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/import_settings/simple.js:2
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/import_progress/import_progress.js:8
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/edit_flyout/overrides.js:6
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/edit_flyout/edit_flyout.js:2
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/analysis_summary/analysis_summary.js:2
x-pack/legacy/plugins/ml/public/file_datavisualizer/components/fields_stats/field_stats_card.js:8
x-pack/legacy/plugins/ml/public/timeseriesexplorer/components/forecasting_modal/forecast_progress.js:6
x-pack/legacy/plugins/ml/public/timeseriesexplorer/components/forecasting_modal/modal.js:4
x-pack/legacy/plugins/telemetry/public/components/__snapshots__/telemetry_form.test.js.snap:2
x-pack/legacy/plugins/uptime/public/components/functional/integration_group.tsx:4
x-pack/legacy/plugins/uptime/public/components/functional/empty_state/__tests__/__snapshots__/empty_state.test.tsx.snap:6
x-pack/legacy/plugins/uptime/public/components/functional/snapshot.tsx:2
x-pack/legacy/plugins/uptime/public/components/functional/charts/duration_chart.tsx:2
x-pack/legacy/plugins/uptime/public/components/functional/charts/checks_chart.tsx:2
x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/condensed_check_list.tsx:2
</details>